### PR TITLE
fix: toaster class

### PR DESCRIPTION
**Pull request by sadeghbarati**
@xiaoluoboding 

Apologize for something that slipped out of my hand :pray:

in `shadcn-vue` we need this

### CODE REVIEW

**Comment by sadeghbarati** [comment]
Can you also add some documents with the Vue `Teleport` component?

**Comment by sadeghbarati** [comment]
@xiaoluoboding 



[this commit](https://github.com/xiaoluoboding/vue-sonner/pull/55/commits/83f27fa5150d681d330671df038c31b55517ce80) is for `shadcn-vue` CLI, `detypes` error while adding `vue-sonner` component

`detypes` is a tool that removes typescript codes from `.vue` and `.ts` files

To resolve `detypes` error I had to merge all those d.ts files generated with `vue-tsc` to one file (`vue-sonner.d.ts`) with `@microsoft/api-extractor`

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "vite",
     "build:docs": "vite build --mode docs",
-    "build:lib": "vite build --mode lib && vue-tsc -p tsconfig.build.json",
+    "build:lib": "vite build --mode lib && pnpm run build-types",
+    "build-types": "vue-tsc -p tsconfig.build.json && api-extractor run",
```
Comment (xiaoluoboding) [inline package.json:10]: Can I change the script to `build:types`

Comment (sadeghbarati) [reply package.json]: @xiaoluoboding done

**Review by xiaoluoboding — APPROVED** [review]
LGTM

**Pull request merged**